conda_recipes/aftereffects-25.0/README.md

```diff
@@ -0,0 +1,45 @@
+# Adobe After Effects 25 conda build recipe
+
+## Creating an archive file for Windows
+
+The Windows installer requires Administrator permissions that are not available in most conda package
+build environments, such as on a Deadline Cloud service-managed fleets. Follow these instructions to
+install Adobe After Effects 25 on a freshly created EC2 instance as Administrator, and create an archive file for use by the conda build recipe. If you have a Windows workstation, you can also do step 3 and 5 without starting an EC2 instance.
+
+1. Launch a fresh Windows Server 2022 instance.
+   1. From the AWS EC2 management console, select the option to Launch instance.
+   2. Enter instance name "Create Windows AE archive".
+   3. Select "Microsoft Windows Server 2022 Base" for the AMI.
+   4. Select an instance type with enough vCPUs and RAM, for example c5.4xlarge has 8 vCPUs and 16 GiB RAM.
+   5. Select "Proceed without a key pair" for the "Key pair (login)" option.
+   6. Make sure that "Allow RDP traffic" is unchecked. We will use SSM port forwarding to avoid sending RDP
+      protocol traffic directly over the internet.
+   7. Set the storage to at least 64 GiB. Adjust other settings as you like, e.g. if you want an encrypted volume of type gp3.
+   8. Select "Launch instance."
+   9. If it asks, select "Proceed without key pair" and proceed with the launch.
+   10. Once it launched, navigate to the instance detail page. Select "Connect," and with "Session manager" selected, again select "Connect."
+       If it says "SSM Agent is not online," you may have to wait a few minutes for it to initialize.
```

Comment:
Vanessa was hitting an issue with this where the SSM agent was not activating despite waiting a very long time. Please sync with her in case these instructions need to be updated with troubleshooting notes.

Reply:
I don't think that is the scope of this recipe. We can link a SSM official guide here but I honestly don't think this is very helpful.